source/features/fit-textareas.tsx

```diff
@@ -20,7 +20,12 @@ function inputListener({target}: Event): void {
 
 function watchTextarea(textarea: HTMLTextAreaElement, {signal}: SignalAsOptions): void {
 	// Disable constrained GitHub feature
-	textarea.classList.replace('js-size-to-fit', 'rgh-fit-textareas');
+	if (textarea.classList.contains('js-size-to-fit')) {
+		textarea.classList.replace('js-size-to-fit', 'rgh-fit-textareas');
+	} else {
+		textarea.classList.add('rgh-fit-textareas');
+	}
```

Comment:
also happy to do this differently, if desired:

1. only 1 line of `textarea.classList.add('rgh-fit-textareas');`
2. add CSS to `.rgh-fit-textareas` class in `source/features/fit-textareas.css` to override the `.js-size-to-fit` styles

Reply:
No conditions are necessary, you can just remove the class (whether it exists or not) and add the other. No "replace" call

Reply:
sounds good:

- 05b69265182fdc4dd73dbad0b988fcb3cf377506